src/logicblocks/event/store/store.py

```diff
@@ -246,13 +254,17 @@ class EventStore:
 
     def __init__(
         self,
-        adapter: EventStorageAdapter,
+        adapter: A,
         logger: FilteringBoundLogger = _default_logger,
     ):
         self._adapter = adapter
         self._logger = logger
 
-    def stream(self, *, category: str, stream: str) -> EventStream:
+    @property
+    def adapter(self) -> A:
+        return self._adapter
```

Comment:
This is implementation detail of `EventStore`, it shouldn't be on the interface.

Reply:
See above - I can remove this and let the user manage keeping a reference to the adaptor